From ee3f3524602c158bec041fdc335403c8ee96ed7b Mon Sep 17 00:00:00 2001
From: "Adrian Parvin D. Ouano" <adrianparvino@gmail.com>
Date: Fri, 27 Apr 2018 17:28:27 +0800
Subject: [PATCH] hardeningDisable for gcc-wrap

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 21e4761..927f533 100644
--- a/Makefile
+++ b/Makefile
@@ -117,7 +117,7 @@ lib/%.o: crt/%.o
 	cp $< $@
 
 tools/gcc-wrap: config.mak
-	printf '#!/bin/sh\nexec gcc -fno-stack-protector -static -D_GNU_SOURCE "$$@" -specs "%s/gcc-wrap.specs"\n' "$(libdir)" > $@
+	printf '#!/bin/sh\nhardeningDisable=all exec gcc -fno-stack-protector -static -D_GNU_SOURCE "$$@" -specs "%s/gcc-wrap.specs"\n' "$(libdir)" > $@
 	chmod +x $@
 
 $(DESTDIR)$(bindir)/%: tools/%
-- 
2.16.2

